src/cljs/athens/electron/db_menu/core.cljs

```diff
@@ -0,0 +1,173 @@
+                                              :last-modified '#inst"2021-09-13T18:35:38.277-00:00"
+                                              :path          "athensresarch.org/company-x"
+                                              :name          "athensresarch.org/company-x"
+                                              :password      "x"
+                                              :remote?       true
+                                              :synced        false}})
+
+
+;; Style
+
+(def dropdown-style
+  {::stylefy/manual [[:.menu {:background (color :background-plus-2)
+                              :color (color :body-text-color)
+                              :border-radius "calc(0.25rem + 0.25rem)" ; Button corner radius + container padding makes "concentric" container radius
+                              :padding "0.25rem"
+                              :display "inline-flex"
+                              :box-shadow [[(:64 DEPTH-SHADOWS) ", 0 0 0 1px rgba(0, 0, 0, 0.05)"]]}]]})
+
+
+(def db-menu-button-style
+  {:color (color :body-text-color :opacity-high)
+   :background "inherit"
+   :padding "0"
+   :border "1px solid transparent"
+   ::stylefy/manual [["&:hover" {:filter "brightness(110%)"}]
+                     ["&:active" {:filter "brightness(90%)"}]
+                     [:&.is-active {:color (color :body-text-color);
+                                    :filter "brightness(90%)"}]
+                     [:.icon {:width "1.75em"
+                              :height "1.75em"}]]})
+
+
+(def current-db-area-style
+  {:background "rgba(144, 144, 144, 0.05)"
+   :margin "-0.25rem -0.25rem 0.125rem"
+   :border-bottom [["1px solid " (color :border-color)]]
+   :padding "0.25rem"})
+
+
+(def current-db-tools-style
+  {:margin-left "2rem"})
+
+
+;; Components
+
+(defn current-db-tools
+  ([{:keys [db]} all-dbs]
+   (let [[key db] db]
+     [:div (use-style current-db-tools-style)
+      (if (:is-remote db)
+        [:<>
+         [button "Import"]
+         [button "Copy Link"]
+         [button "Remove"]]
+        [:<>
+         [button {:onClick #(dialogs/move-dialog!)} "Move"]
+         ;; [button {:onClick "Rename"]
+         [button {:onClick #(if (= 1 (count all-dbs))
+                              (js/alert "Can't remove last db from the list")
+                              (dispatch [:db-picker/delete-db (:path db)]))}
+          "Delete"]])])))
+
+
+(defn db-menu
+  []
+  (r/with-let [ele (r/atom nil)]
+              (let [current-db-path  @(subscribe [:db/filepath])
+                    all-dbs          @(subscribe [:db-picker/all-dbs])
+                    #_#_#_#_active-db        (first (filter #(= (:path %) current-db-path) all-dbs))
+                    inactive-dbs     (filter #(not= (:path %) current-db-path) all-dbs)
+                    active-db (first all-dbs)
+                    inactive-dbs (next all-dbs)
```

Comment:
These functions are not how to actually determine if the db is active or not. Must compare `all-dbs` with the currently active db, which is currently determined by `:db/filepath`.